### PR TITLE
mpd: update build recipe

### PR DESCRIPTION
**Pull request by tts2k**

### CODE REVIEW

File: media-sound/mpd/mpd-0.20.12.recipe [hunk 1]
```diff
@@ -0,0 +1,158 @@
+SUMMARY="The Music Player Daemon"
+DESCRIPTION="Music Player Daemon (MPD) is a flexible, powerful, server-side \
+application for playing music. Through plugins and libraries it can play a \
+variety of sound files while being controlled by its network protocol."
+HOMEPAGE="http://www.musicpd.org/"
```
Comment (fbrosson) [inline media-sound/mpd/mpd-0.20.12.recipe]:
```diff
-HOMEPAGE="http://www.musicpd.org/"
+HOMEPAGE="https://www.musicpd.org/"
```

File: media-sound/mpd/mpd-0.20.12.recipe [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+	#lib:libao$secondaryArchSuffix >= 4
+	#lib:libaudiofile$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libavformat$secondaryArchSuffix
+	lib:libavutil$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcdio$secondaryArchSuffix
+	#lib:libcdio_paranoia$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	#lib:ldns_sd$secondaryArchSuffix
```
Comment (fbrosson) [inline media-sound/mpd/mpd-0.20.12.recipe]:
```diff
-	#lib:libcdio_paranoia$secondaryArchSuffix
+#	lib:libcdio_paranoia$secondaryArchSuffix
 	lib:libcurl$secondaryArchSuffix
-	#lib:ldns_sd$secondaryArchSuffix
+#	lib:ldns_sd$secondaryArchSuffix
```
(similar changes elsewhere)

File: media-sound/mpd/mpd-0.20.12.recipe [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+	cmd:autoconf
+	#cmd:avahi_client$secondaryArchSuffix
+	cmd:awk
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+#	cmd:ld$secondaryArchSuffix
+#	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:m4
```
Comment (fbrosson) [inline media-sound/mpd/mpd-0.20.12.recipe]:
```diff
-	cmd:make
 	cmd:m4
+	cmd:make
```

**Comment by extrowerk** [comment]
The 0.2.13 just released. Can you check and update your recipe please?

**Comment by tts2k** [comment]
Sorry for late reply but sure !

**Comment by extrowerk** [comment]
Looks great, but sadly i have no time right now to test it. Which platform have you tested? Does it packaged fine?

**Comment by tts2k** [comment]
Tested and packaged fine on x86_64, not sure about other platforms tho.

**Pull request merged**